embedded-graphics/src/image/image16bpp.rs

```diff
@@ -94,9 +96,7 @@ where
                 self.y += 1;
             }
 
-            if current_pixel[0] >= 0 && current_pixel[1] >= 0 {
-                break Pixel(current_pixel.to_unsigned(), C::from_raw_data(data));
-            }
+            break Pixel(current_pixel, C::from_raw_data(data));
```

Comment:
I think this is also a breaking change - images no longer self-check if pixels are greater than zero, so it's up to the display driver, right? A changelog note would be appreciated so I don't forget when I'm writing it up.

Reply:
Yes, i've added a changelog note, but #155 also describes this change.

Reply:
Thanks, good point on #155 but all these changes are tricky to keep track of (for me at least) so I'd prefer to have the redundancy of a note in the changelog.